app/assets/javascripts/application.js

```diff
@@ -1 +1,6 @@
+// This file needs to be in sync with webpack.config.prod.js. The first file
+// should contain idseq-written JS and the second file should contain JS from
+// node_modules.
+
 //= require bundle.min
+//= require 1.bundle.min
```

Comment:
Is it possible to specify more friendly names? Namely for the second one? 
We could do `main.bundle.min` and `third-party.bundle.min`...

Reply:
I tried to do this for about 15min and I gave up in frustration. I'll try again now 😉